src/Cody.VisualStudio/Client/AgentClient.cs

```diff
@@ -127,12 +128,7 @@ private void OnAgentDisconnected(object sender, int exitCode)
             //to many calls to sentry
             //else log.Error($"The agent connection unexpectedly ended with code {exitCode}.");
 
-            if (options.RestartAgentOnFailure && exitCode != 0 && !VsShellUtilities.ShutdownToken.IsCancellationRequested)
```

Comment:
`RestartAgentOnFailure` in the options is unused now. pls remove it

Reply:
OK, removed.